editor/editor_help.cpp

```diff
@@ -329,6 +330,7 @@ void EditorHelp::_class_desc_select(const String &p_select) {
 		OS::get_singleton()->shell_open(p_select);
 	} else if (p_select.begins_with("^")) { // Copy button.
 		DisplayServer::get_singleton()->clipboard_set(p_select.substr(1));
+		EditorToaster::get_singleton()->popup_str(TTR("Copied to clipboard."), EditorToaster::SEVERITY_INFO);
```

Comment:
Seems fine, though I can think of plausible alternatives:
```suggestion
		EditorToaster::get_singleton()->popup_str(TTR("Example copied to clipboard."), EditorToaster::SEVERITY_INFO);
```
or
```suggestion
		EditorToaster::get_singleton()->popup_str(TTR("Code example copied to clipboard."), EditorToaster::SEVERITY_INFO);
```
or
```suggestion
		EditorToaster::get_singleton()->popup_str(TTR("Code snippet copied to clipboard."), EditorToaster::SEVERITY_INFO);
```
If this same UX pattern of a toast when copying is to be repeated elsewhere, it's useful to distinguish the fact that it's code that is copied. On the other hand, maybe it's good to keep the number of translatable strings lower with a generic message like the current one.

Reply:
I'm going to say "Code snippet" to keep it generic.